src/main/resources/application.yaml

```diff
@@ -221,4 +221,4 @@ process-payments:
 use-wrapping-zip: true
 
 ocr-validation-max-retries: 2
-ocr-validation-delay-retry-ms: 120000
+ocr-validation-delay-retry-sec: 120
```

Comment:
a bit small window. especially when only 2 retries i would expect bigger window and/or even apply some geometric progression to make window longer after each retry

Reply:
p.s. can think about it and apply in another pr ;)